modules/ossm-installation-activities.adoc

```diff
@@ -2,15 +2,33 @@
 //
 // * service_mesh/v1x/preparing-ossm-installation.adoc
 // * service_mesh/v2x/preparing-ossm-installation.adoc
-// * post_installation_configuration/network-configuration.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="ossm-installation-activities_{context}"]
-= Operator overview
+= Service Mesh Operators overview
 
-{SMProductName} requires the following Operators:
+{SMProductName} requires the use of the {SMProductName} Operator which allows you to connect, secure, control, and observe the microservices that comprise your applications. You can also install other Operators to enhance your service mesh experience.
 
-* *OpenShift Elasticsearch* - (Optional) Provides database storage for tracing and logging with the {JaegerShortName}. It is based on the open source link:https://www.elastic.co/[Elasticsearch] project.
-* *{JaegerName}* - Provides distributed tracing to monitor and troubleshoot transactions in complex distributed systems. It is based on the open source link:https://www.jaegertracing.io/[Jaeger] project.
-* *Kiali Operator (provided by Red Hat)* - Provides observability for your service mesh. You can view configurations, monitor traffic, and analyze traces in a single console. It is based on the open source link:https://www.kiali.io/[Kiali] project.
-* *{SMProductName}* - Allows you to connect, secure, control, and observe the microservices that comprise your applications. The {SMProductShortName} Operator defines and monitors the `ServiceMeshControlPlane` resources that manage the deployment, updating, and deletion of the {SMProductShortName} components. It is based on the open source link:https://istio.io/[Istio] project.
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+The following Operator is required:
+
+{SMProductName} Operator:: Allows you to connect, secure, control, and observe the microservices that comprise your applications. It also defines and monitors the `ServiceMeshControlPlane` resources that manage the deployment, updating, and deletion of the {SMProductShortName} components. It is based on the open source link:https://istio.io/[Istio] project.
```

Comment:
[ISG] "open-source" when modifying a noun directly (here and others like it in PR)